aries-core/src/js/components/external/stories/RangeSelector.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { RangeSelector } from 'aries-core';
```

Comment:
why is this story is on the `external` package while RangeSelector is part of aries-core?

Reply:
Good catch. Thanks Shimi. Let me update where that file is living.